src/main/java/edu/gatech/cs2340/trydent/GameObject.java

```diff
@@ -25,7 +25,7 @@
 
     private String name = "Untitled GameObject";
 
-    private Group fxNode;
+    protected Group fxNode;
```

Comment:
I would prefer it be private, with a protected getter, unless subclasses need to assign to fxNode. The reason it's hidden a bit is that our parent/child tree is externally simplified but internally complicated, so I tried to separate the distinction as much as possible.